src/components/Layout/Layout.tsx

```diff
@@ -167,6 +169,36 @@ export const Layout = ({
       menuButtonRef?.current?.focus();
     }
   };
+  let menu = <Menu
+    currentPlatform={currentPlatform}
+    rootMenuNode={rootMenuNode}
+    menuTitle="How Amplify works"
+    menuHref={{
+      pathname: `/[platform]`,
+      query: { platform: currentPlatform }
+    }}
+  />
+  if (isGen2) {
+    menu = <Menu
+      rootMenuNode={rootMenuNode}
+      menuTitle="How Gen2 Amplify works"
+      menuHref={{
+        pathname: `/gen2`
+      }}
+    />
+  } else if (isPrev) {
+    let prevNode = findDirectoryNode('/[platform]/prev');
+    menu = <Menu
+      currentPlatform={currentPlatform}
+      rootMenuNode={prevNode}
+      menuTitle="How Amplify works"
+      menuHref={{
+        pathname: `/[platform]/prev`,
+        query: { platform: currentPlatform }
+      }}
```

Comment:
These can be removed too, right? I don't think you're using them inside `Menu` because it doesn't accept those props anymore